pkg/chunk/table_manager.go

```diff
@@ -202,36 +207,65 @@ func (m *TableManager) bucketRetentionLoop() {
 // SyncTables will calculate the tables expected to exist, create those that do
 // not and update those that need it.  It is exposed for testing.
 func (m *TableManager) SyncTables(ctx context.Context) error {
-	expected := m.calculateExpectedTables()
-	level.Info(util.Logger).Log("msg", "synching tables", "num_expected_tables", len(expected), "expected_tables", len(expected))
+	for i, client := range m.clients {
+		if client == nil {
+			continue
+		}
+		lastClient := i == len(m.clients)-1
+		expected := m.calculateExpectedTables(i)
+		if lastClient {
+			level.Info(util.Logger).Log("msg", "synching tables", "num_expected_tables", len(expected), "expected_tables", len(expected))
```

Comment:
What is the difference between `num_expected_tables` and `expected_tables` in these two log statements?

Reply:
Ah I already opened a PR to fix it upstream before looking at this comment https://github.com/cortexproject/cortex/pull/1754, they are the same. Noticed this log when running Loki.

Will fix it here too.